tesseract/tesseract_scene_graph/include/tesseract_scene_graph/resource_locator.h

```diff
@@ -0,0 +1,65 @@
+#ifndef TESSERACT_SCENE_GRAPH_RESOURCE_LOCATOR_H
+#define TESSERACT_SCENE_GRAPH_RESOURCE_LOCATOR_H
+
+#include <tesseract_common/resource.h>
+#include <tesseract_scene_graph/allowed_collision_matrix.h>
+#include <tesseract_scene_graph/graph.h>
+#include <tesseract_scene_graph/parser/srdf_parser.h>
+#include <functional>
```

Comment:
I would wrap this in the macro to prevent warnings generated form system dependencies.